cogs5e/models/character.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, owner: str, upstream: str, active: bool, sheet_type: str, imp
 
         # main character info
         self._description = description
-        self._image = image
+        self._image = image or ''
```

Comment:
Instead of modifying the character data, a better pattern is to put this in the `image` property getter